### ISSUE TITLE
Filling out the individual reports for Milestone 2

### ISSUE THREAD

**Issue by ege-kaya** [bounswe/2021SpringGroup2#72]
Everyone should start filling out their individual reports for the Milestone 2 submission in accordance with the guidelines provided on Moodle. We should preferably have made substantial progress by Tuesday.

**Comment by berkaydoner** [comment]
After I read the deliverables for this report, I decided to have following headers in my report:
- [ ] Executive Summary
- [ ] Learning Phase 
- [ ] API 
- [ ] Code Review
- [ ] Pull Requests
- [ ] Issues

**Comment by berkaydoner** [comment]
I provided the necessary headers in my report.
* In the executive summary, I provided a simple summary of our practice app.
* In the learning phase, I explained how I learned about the tools I used during the implementation phase such as Flask, PostgreSQL.
* In the API part, I explained the external API's I used and why I used them. I provided the method and its unit tests and explained them for each of the three functions I implemented. I also wrote my contributions to the database and finally discussed about the difficulties I encountered.
* In the code review part, I listed all the pull requests I provided a review. For each, I explained my review and what it changed in the code.
* In the pull requests section, I listed all the pull requests I opened. For each, I explained the feedback I got and how I changed my code to apply requested changes.
* In the issues part, I provided links to the issues I have been assigned and I created. For each, I gave a summary of the issue and its purpose.

I think I have explained all my efforts and my report is complete.

**Comment by ibrahimbayat** [comment]
- In my individual report, I explained our project and my work related to project in executive summary part. 

- During the project, I learned many things and made research about them. I provided some URI related to research sources.

- I mentioned my challenges while working on the project. 

- I explained my functions, unit tests and mentioned their related issues and pull requests. I added what did I change according to feedbacks in my pull requests. Also, I tried to add their sample calls and outputs as far as I can.

- I added my reviews and its conversations. Then what happened after my reviews to these pull requests. 

- Finally I sended my zip file with the report and source code files which I contributed.
